conda_smithy/templates/README.md.tmpl

```diff
@@ -1,3 +1,5 @@
+{% set channel_name = channels.targets[0][0] %}
```

Comment:
Is this the way to do it? It causes additional blank lines. How do we avoid that?